backend/src/game.rs

```diff
@@ -16,10 +19,33 @@ impl Game {
             field: Grid::new(row_n, col_n),
         }
     }
+
+    pub fn set_unit(&mut self, x: u32, y: u32, unit: Unit) -> Result<(), &str> {
```

Comment:
Change order to (y, x)

Reply:
Better also rename vars to `row`, `col` then.